src/routes/home/components/HomePage.js

```diff
@@ -30,3 +30,9 @@ internals.HomepageContainer = Styled.div`
     align-self: center;
     margin: auto;
 `;
+
+internals.WelcomeMessage = Styled(Typography).attrs({ variant: 'h4', align: 'center' })`
+
+    // Example leveraging the mui theme from inside a styled-component
+    color: ${(props) => props.theme.palette.duckYellow.main};
```

Comment:
Does this cause tests to fail?  I could imagine `duckYellow.main` not existing on the default theme in a way that would cause a hard failure.